### PR TITLE
#164694210 Integrate TravisCI with readme badge

### PR DESCRIPTION
**Pull request by Rhytah**
#### What does this PR do?
Have TravisCI build badge in the readme file.

#### Description of Task to be completed

- [x] Write a `.travis.yml` configuration file.

- [x] Add TravisCI integration hooks in Github repository

- [x] Sync Github repository on TravisCI domain

- [x] Attach build status badge within the repository `README.md` file

#### How can this manually be tested?
On opening Github repository [repo](https://github.com/mirest/ah-haven-space-sprinters/tree/ch-Integrate-TravisCI-164694210)

- You should see a badge descriptive of `build` in the readme
#### What are the relevant pivotal tracker stories?
#164694210

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+[![Build Status](https://travis-ci.org/mirest/ah-haven-space-sprinters.svg?branch=ch-Integrate-TravisCI-164694210)](https://travis-ci.org/mirest/ah-haven-space-sprinters)
```
Comment (kimbugp) [inline README.md]: Nice work @Rhytah. Could you adjust the badge to show the build status for `develop`.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+env: 
+
+  -DJANGO=1.8 DB=postgresql
+install: 
+
+  - pip install -r requirements.txt
+before_script:
+  - psql -c 'create database testdb;' -U postgres
+
```
Comment (kimbugp) [inline .travis.yml:20]: could you cater for database migrations in the before script

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+script: 
+  - pytest
```
Comment (kimbugp) [inline .travis.yml]: add a new line at the end of the line

File: .travis.yml [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+services: 
+
+  - postgresql
+
+env: 
+
+  -DJANGO=1.8 DB=postgresql
+install: 
+
+  - pip install -r requirements.txt
```
Comment (kimbugp) [inline .travis.yml:15]: add pytest to the requirements file

**Comment by kimbugp** [comment]
@Rhytah , could you also squash your commits to have one

File: README.md (hunk 1 above)
Comment (Rhytah) [reply README.md]: I have adjusted this. Thank you.

File: requirements.txt [hunk 5]
```diff
@@ -1,6 +1,22 @@
+atomicwrites==1.3.0
```
Comment (kimbugp) [inline requirements.txt]: also, just remove packages you haven't by added yourself.
Only add the necessary ones

**Comment by kimbugp** [comment]
@Rhytah, Could you please change the commit to be in the present tense. Look at the wiki https://github.com/mirest/ah-haven-space-sprinters/wiki/Code-Conventions#message-body for context

**Review by kimbugp — APPROVED** [review]

**Pull request merged**